src/current/v24.3/create-logical-replication-stream.md

```diff
@@ -0,0 +1,148 @@
+`logical_replication_options` | Options to modify the behavior of the LDR stream.
+
+## Options
+
+Option | Description
+-------+------------
+`cursor` | Emits any changes after the specified timestamp. LDR will not perform an initial backfill with the `cursor` option, it will stream any changes after the specified timestamp. The LDR job will encounter an error if you specify a `cursor` timestamp that is before the configured [garbage collection]({% link {{ page.version.version }}/architecture/storage-layer.md %}#garbage-collection) window for that table. Use `cursor` to [Bootstrap LDR from a backup](#bootstrap-ldr-from-a-backup) to avoid the initial backfill of data onto the destination cluster from source cluster.
+`discard` | Ignore [TTL deletes]({% link {{ page.version.version }}/row-level-ttl.md %}) in an LDR stream. Use `discard = ttl-deletes`. **Note**: To ignore row-level TTL deletes in an LDR stream, it is necessary to set the [`ttl_disable_changefeed_replication`]({% link {{ page.version.version }}/row-level-ttl.md %}#ttl-storage-parameters) storage parameter on the source table. Refer to the [Ignore row-level TTL deletes](#ignore-row-level-ttl-deletes) example.
+`label` | Tracks LDR metrics at the job level. Add a user-specified string with `label`. Refer to [Metrics labels]({% link {{ page.version.version }}/logical-data-replication-monitoring.md %}#metrics-labels).
+`mode` | Determines how LDR replicates the data to the destination cluster. Possible values: `immediate`, `validated`. For more details refer to [LDR modes](#ldr-modes).
+
+## LDR modes
+
+_Modes_ determine how LDR replicates the data to the destination cluster. There are two modes:
+
+- `immediate` (default): Attempts to replicate the changed row directly into the destination table, without re-running constraint validations or triggers. It does not support writing into tables with [foreign key]({% link {{ page.version.version }}/foreign-key.md %}) constraints.
+- `validated`: Attempts to apply the write in a similar way to a user-run query, which would re-run all constraint validations or triggers relevant to the destination table(s). It will potentially reject the change if it fails rather than writing it, which will cause the row change to enter the DLQ instead.
+
+## Examples
+
+To start LDR, you must run the `CREATE LOGICAL REPLICATION STREAM` statement from the **destination** cluster. Use the[ fully qualified table name(s)]({% link {{ page.version.version }}/sql-name-resolution.md %}#how-name-resolution-works) and ensure that the source and destination table names are identical. The following examples show statement usage with different options and use cases.
+
+### Start an LDR stream
+
```

Comment:
i think this page could have so more commentary around the order in which you start a bidi stream. Perhaps this is coming later. But at a high level:
- LDR only supports starting with two empty tables, or one non empty table. we do not support starting from two non-empty tables.
- if you're starting with one non-empty table: start the LDR stream from empty to non empty first, i.e.  run CREATE LDR from the non-empty table first.

Reply:
OK, I've added a small section here. "Bidirectional LDR" I think I might move this though to the overview page, and do some link references once the first few PRs have merged.